conans/test/integration/toolchains/test_content.py

```diff
@@ -0,0 +1,301 @@
+import textwrap
+
+lib_h = textwrap.dedent("""
```

Comment:
This has already been factored out. Please check the new ``gen_function_cpp``, ``gen_function_h``.
Please do not include test framework refactors in this PR, focus only on adding intel tests.

Reply:
okay, now I use `gen_function_cpp`